Assets/MRTK/Core/Utilities/WebRequestRest/Response.cs

```diff
@@ -16,12 +16,14 @@ public struct Response
         /// <summary>
         /// Response body from the resource.
         /// </summary>
-        public string ResponseBody { get; }
+        public string ResponseBody => responseBodyAction?.Invoke();
```

Comment:
Will this now cause the conversion every time this property is accessed, or does the `downloadHandler` cache it? If it'll now be converted on every access, could this class cache it on first access in case it's called multiple times?

Reply:
I guess it's getting converted everytime. At least nothing else is said about this in the docs:
https://docs.unity3d.com/ScriptReference/Networking.DownloadHandler.GetText.html